pkgs/applications/science/robotics/mission-planner/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchurl, makeDesktopItem, makeWrapper, unzip, mono6 }:
+
+let
+  pname = "mission-planner";
+  desktopItem = makeDesktopItem {
+    name = pname;
+    exec = pname;
+    icon = pname;
+    comment = "MissionPlanner GCS & Ardupilot configuration tool";
+    desktopName = "MissionPlanner";
+    genericName = "Ground Control Station";
+  };
+in stdenv.mkDerivation rec {
+  inherit pname;
+  version = "1.3.74";
+
+  src = fetchurl {
+    url = "https://firmware.ardupilot.org/Tools/MissionPlanner/MissionPlanner-${version}.zip";
+    sha256 = "1cgpmsmmnbzw1lwsdafp8yklk1rwc61yf12vc1ahcc6bl7q2385x";
+  };
+
+  nativeBuildInputs = [ makeWrapper mono6 unzip ];
+  sourceRoot = ".";
+
+  AOT_FILES = [ "MissionPlanner.exe" "MissionPlanner.*.dll" ];
+
+  buildPhase = ''
+    for file in $AOT_FILES
+    do
+      mono --aot $file
+    done
+  '';
+
+  installPhase = ''
+    # create target directoryies
+    mkdir -p $out/{bin,opt/mission-planner}
```

Comment:
```suggestion
    mkdir -p $out/bin $out/opt/mission-planner
```

Reply:
Why is more repetition better?

Reply:
Using ``{..,..}`` is nice if only a small part of a path changes like the last directory in a 5 deep nested path. I don't think it makes the code more readable if ``$out/`` is the only repeating part.

Reply:
I disagree on this one as well, but it likely boils down to personal preference. Is it a pressing issue to you having this changed?